docs/release_notes/release_notes.md

```diff
@@ -9,52 +9,58 @@ has_children: true
 # Release Notes
 
 
-## Version 3.1.0 (March 2023)
+## Version 4.0.0 (May 2023)
 
-* ### Monitor Linux Groups & Users (Linux/Unix)
-    * Added savedsearch and dashboard panels to track the group and user changes for the Linux environment.
+* ### Notable Event Assignment
+    * User can assign a notable event to user and update the notable event status based the investigation from the Forensic dashboard.
+    * Added SOC Dashboard to get overview of notable events by the alert, assignee, status and severity.
+    * Updated Overview dashboard to filter the result based on the Status value.
 
-* ### Sophos Firewall Integration
-    * Mapped Sophos Firewall Traffic data with the Network_Traffic Model.
-    * Added new Sophos Firewall dashboard
-    ![alt]({{ site.baseurl }}/assets/sophos_firewall.png)
-    * To onboard the Sophos Firewall data, Refer [Sophos Firewall Data Onboarding]({{ site.baseurl }}/data_onboarding/network_devices/sophos_firewall/)
+![alt]({{ site.baseurl }}/assets/notable_event_assignment.png)
+
+* ### New Cyences Settings Page
+    * User can easily enable/disable the products as per their Splunk environment from the Product Setup page to enable/disable savedsearch and dashboards accordingly.
+    * All the old configurations are migrated to the new Cyences App Configuration Page. 
+
+![alt]({{ site.baseurl }}/assets/data_source_macros.png)
 
-* ### Inbound Vulnerable Traffic Alerts
-    * Added new "Network Compromise - Inbound Vulnerable Traffic" to alerts the incoming traffic on the vulnerable port.
 
 * ### Enhancements
-    * VPN
-        * Added more filters to filter results by Users (Multiselect), Host, Public IP Address and Private IP Address.
+    * Enhanced MultiSelect functionality
+        * MultiSelect input will automatically select/unselect All option based on the user selection in all the dashboards.
     
-    * Bruteforce Alerts
-        * Improved search query to show the count for src, user and dest fields.
-        * Reduced severity from critical to high for windows event if src field value is unknown.
+    * Support for email messagetrace event using O365 addon 
+        * Updated Email related alerts to support email messagetrace event collected using the O365 addon.
 
-    * AD Password Change Alerts and Dashboard
-        * Improved search query to show the count for ComputerName, status and Actor fields.
-        * Added Probable Sources field to show the IP Address detail.
+    * Reduced severity level for the already blocked events for Palo High Threat alert.
 
+    * Windows - Windows Process Tampering Detected alert
+        * Added internal filter macro to reduce the false positives.
 
-* ### Bug Fixes
-    * Fixed the filter issue for the Windows Patch dashboard.
+    * New alerts for Successful Login from new location
+        * O365 - Login Failure From New Location Due To Multi Factor Authentication
+        * O365 - Successful Login From New Location
```

Comment:
From New Country

Reply:
include VPN related alert as well.
In the bracket specify (The alert is vastly improved to make it login from a new Country instead of a static country name. This also reduces false positives drastically. This also applies to the other two newly added alerts as well.)